.prettierrc

```
@@ -0,0 +1 @@
+{}
```

Ignorable suggestion: it looks like a lot of the original code was written with single quotes. Prettier doesn’t care what you use and you can configure it; this might reduce a _little_ bit of noise. But ultimately it’s stylistic preference that doesn’t matter.

```suggestion
{
  "singleQuote": true
}
```